aslam_offline_calibration/kalibr/scripts/run_kalibr_tarfile.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+FOLDER=$1
+
+if [ ! -d "$FOLDER" ]; then
+  echo "Folder $FOLDER does not exist!! Please check and run again!"
+  exit
+fi
+
+echo "Folder with tar file : "
+echo $FOLDER
+
+rosrun kalibr kalibr_data_import.sh $FOLDER
+
+echo "Imported data from tar file!"
+
+rosrun kalibr kalibr_bagcreater --folder $FOLDER/calibration_params \
+                                --output-bag $FOLDER/kalibr_data.bag
+
+echo "Created Kalibr input image bag file!"
+
+KALIBR_LOCATION=`rospack find kalibr`
+
+echo "Location of Kalibr package : "
+echo $KALIBR_LOCATION
+
+echo "Now Calibrating!!"
```

Comment:
This is the same as `echo -e "Now Calibrating!!\n\n"` or `echo -ne "Now Calibrating!!\n\n\n"`

Consider changing it (optional).